### PR TITLE
feat: initial impl of momento keyv

### PR DESCRIPTION
**Pull request by eaddingtonwhite**

### CODE REVIEW

File: README.template.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+## Installation
+
+```shell
+npm install --save @gomomento-poc/node-keyv-adaptor
+```
+
+## Usage
+
+```ts
```
Comment (cprice404) [inline README.template.md:21]: FYI with the new v2 template engine you can just use an `include` tag to pull in as many code files as you want:

https://github.com/momentohq/client-sdk-javascript/blob/main/README.template.md?plain=1#L16

not a blocker for this pr but just letting you know, that way we can make sure ci runs the code

**Review by cprice404 — APPROVED** [review]
would love to add an int test layer that validates our behavior vs some other backend using a docker container, but not a blocker for this PR.

@malandis should look at this too but I am okay with you merging and he can weigh in after if you have someone you're trying to share this with.

**Comment by eaddingtonwhite** [comment]
@cprice404 this is more of adapter specifically for momento for keyv vs wrapper around redis so I dont think having integration test w/ different backends makes as much sense for this lib. Also have no way to configure a redis run mode for this version of adapator they would just use redis-adapter directly.
https://github.com/jaredwray/keyv/tree/main/packages/redis

File: README.template.md (hunk 1 above)
Comment (eaddingtonwhite) [reply README.template.md:21]: Ill do in follow up PR good call!

**Pull request merged**